README.md

```diff
@@ -1,2 +1,88 @@
-# sbom-action
-GitHub Action for creating software bill of materials using Syft.
+# Anchore SBOM Action
+
+A GitHub Action for creating a software bill of materials (SBOM)
+using [Syft](https://github.com/anchore/syft).
+
+## Basic Usage
+
+```yaml
+- uses: anchore/sbom-action@v1
+```
+
+By default, this action will execute a Syft scan in the workspace directory
+and upload a workflow artifact SBOM in SPDX format. It will also detect
+if being run during a `release` and attach the SBOM
+as a release asset.
+
+## Example Usage
+
+### Scan a docker image
+
+Use the `image` parameter
+
+```yaml
+- uses: anchore/sbom-action@v1
+  with:
+    image: example/image_name
+```
+
+### Scan a specific directory
+
+Use the `path` parameter, relative to the repository root
+
+```yaml
+- uses: anchore/sbom-action@v1
+  with:
+    path: ./build/
+```
+
+### Attach a combined SBOM
```

Comment:
I think we should update this to `Attache Multiple SBOM` until we get the functionality in that builds the `MOASBOM`